### PR TITLE
feat: port the front end sample projects

### PR DESCRIPTION
**Pull request by ieahleen**
Closes https://github.com/freeCodeCamp/freeCodeCamp/issues/46173

### CODE REVIEW

File: apps/markdown-previewer/index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.23.0/themes/prism.min.css"
+    />
+    <title>FCC : Random Quote Machine</title>
+  </head>
+  <body>
+    <div id="app"></div>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/marked/2.0.3/marked.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.23.0/prism.min.js"></script>
+    <script src="script.js"></script>
```
Comment (ShaunSHamilton) [inline apps/markdown-previewer/index.html:19]: Why not include React here?

Comment (ieahleen) [reply apps/markdown-previewer/index.html:27]: ok included

**Review by moT01 — APPROVED** [review]
This LGTM @ieahleen 🎉 

The only thing I was wondering about is if we should include the `bundle.js` files in these projects. The pinterest clone is the only other project (I think) with a bundle, and it's not included there. So I wonder if we ran the build command manually on the VM for that or what - I don't see any build commands included when running `start`, so that would be my guess. So if we don't include the bundles here, we would have to do that or include a build with the start command. I don't really mind including the files here, they're only a few kb's and these projects probably won't be changing much, so we won't need to update them - probably ever. It doesn't seem conventional, but I'm fine with it. Any thoughts on that @ieahleen @ShaunSHamilton @ojeytonwilliams?

**Comment by ieahleen** [comment]
The `bundle.js` files are created when `npm install` or `npm run build` is run globally, so yeah, I will remove those, you are right

**Comment by ieahleen** [comment]
I created the `.gitignore` file for each project

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Pull request merged**